src/components/new-request/search.vue

```diff
@@ -381,6 +406,15 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.scrollTo('namerequest-sbc-header')
   }
 
+  /** Fetch businesses with business lookup depending on the Action selected */
+  get fetchActiveOrHistorical (): String {
+    if (this.isRestoration) {
+      return 'HISTORICAL'
+    } else {
+      return 'ACTIVE'
+    }
+  }
```

Comment:
Please rename this to "lookup" something, since business-fetch is something else.

Also, how about using a ternary here? 😉